app.js

```diff
@@ -31,11 +34,15 @@ function showBucketlistOverzicht(req, res) {
     res.render('pages/bucketlist/bucketlistOverzicht', {title: 'bucketlist'});
   };
 
+
   //function render bucketlistResultaat page
 function showBucketlistResultaat(req, res) {
     res.render('pages/bucketlist/bucketlistResultaat', {title: 'bucketlistoverzicht'}, {interestView: data});
   };
-  
+
+function saveBucketlist(req, res) { 
+  console.log(req.body);
+}
```

Comment:
If we were to add `console.log` in the `develop` branch, our console would be filled with these test logs 😖. Maybe use a comment here if there is something that is supposed to be added later.